app/views/main_content_panel.py

```diff
@@ -983,7 +982,6 @@ def _do_refresh(self, is_initial: bool = False) -> None:
         EventBus().do_save_button_animation_stop.emit()
         # If we are refreshing cache from user action
         if not is_initial:
-            self.mods_panel.list_updated = False
```

Comment:
Was this just unused?

Reply:
As far as I can tell yeah. It's not used anywhere, including other files.